src/test/java/com/bakdata/common_kafka_streams/integration/CleanUpTest.java

```diff
@@ -215,6 +214,65 @@ void shouldDeleteKeySchema(final SoftAssertions softly)
                 .contains(this.app.getInputTopic() + "-key");
     }
 
+    @Test
```

Comment:
Please add a test for a topology with a non-internal topic in between input and output

Reply:
There is a non-internal topic used by `through` in the `ComplexTopologyApplication`. Is there a different use case that should be tested?
I merged the schema deletion tests for non-output topics to make it a little clearer and  to reduce the run time.

Reply:
That's totally fine, good job